### PR TITLE
Update America First Credit Union

### PR DESCRIPTION
**Pull request by morksmail**
Update website URL to the one they redirect to, add TOTP.

### CODE REVIEW

File: entries/a/americafirst.com.json [hunk 1]
```diff
@@ -3,9 +3,10 @@
     "domain": "americafirst.com",
     "tfa": [
       "sms",
-      "email"
+      "email",
+      "totp"
```
Comment (hkamran80) [inline entries/a/americafirst.com.json:7]: I don't see any mention of TOTP. Please attach some screenshots of the MFA setup process, login process, or management screen. Please blur or crop out any personal or sensitive information.

Comment (morksmail) [reply entries/a/americafirst.com.json:7]: I don't either, but I set it up this week and am using it.

Comment (hkamran80) [reply entries/a/americafirst.com.json:7]: Please attach some screenshots of the MFA setup process, login process, or management screen. Please blur or crop out any personal or sensitive information.

**Comment by Carlgo11** [comment]
As the requested changes haven't been resolved yet I'm going to label this PR as inactive.
> [!Warning]
> Unless there's activity within the next 7 days I'll close the PR.

**Pull request closed without merging**